File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>


int count_letters(string text);
int count_words(string text);

int main(void)
{
    //prompt user for text
    string text = get_string("Text: ");
    int letters = count_letters(text);
    int words = count_words(text);

    printf("%i letters\n%i words\n", letters, words);

}

    //count letters
int count_letters(string text)
{
    int n_letters = 0;

    for (int i = 0, len = strlen(text); i < len; i++)
    {
        if (islower(text[i]) || isupper(text[i]))
        {
            n_letters++;
        }
    }
    return n_letters;
}


    //count words
int count_words(string text)
{
    int n_words = 0;

    for (int i = 0, len = strlen(text); i< len; i++)
    {
        if (isspace(text[i]) || \0)
        {
            n_words++;
        }
    }
    return n_words;
}




    //count sentences
    //print "Grade: (X)"; "Grade: 16+" if higher than 16; "Before grade 1" if lower than 1

File: sentimental-readability/readability.py
# TODO
from cs50 import get_string

# prompt user for text
def get_text():
    n = get_string("text: ")
    return n

# return letters, words, sentences
def lts(text):
    letters = count_letter(text)
    words = count_word(text)
    sentences = count_sentence(text)
    return(letters, words, sentences)

# calculate grade
def calcul_grade(letters, words, sentences):
    L = letters / words * 100
    S = sentences / words * 100
    grade = in(0.0588 * L - 0.296 * S - 15.8)
    return grade


# print grade
def print_grade(grade):
    if grade >= 16:
        print("Grade 16+")
    elif grade < 1:
        print("Before Grade 1")
    else:
        print(grade)

# func count letter
def count_letter(text):
    nb = 0
    length = len(text)

    for i in range(length):
        if 


# func count word

# func count sentence
